pkg/controller/cheinstallation/che.go

```diff
@@ -45,10 +45,10 @@ func NewSubscription(ns string) *olmv1alpha1.Subscription {
 			Labels:    toolchain.Labels(),
 		},
 		Spec: &olmv1alpha1.SubscriptionSpec{
-			Channel:                "stable",
-			Package:                "eclipse-che",
-			StartingCSV:            "eclipse-che.v7.2.0",
-			CatalogSource:          "community-operators",
+			Channel:                "latest",
+			Package:                "codeready-workspaces",
+			StartingCSV:            "crwoperator.v2.0.0",
+			CatalogSource:          "redhat-operators",
```

Comment:
I have also switched into on of my ongoing PR but this switch can be done with this PR :+1: 

I have observed that, when you manually install CRW operator, it will set install_plan attribute to automatic.  Please add it to this PR .. for ref: https://github.com/codeready-toolchain/toolchain-operator/pull/17/files?file-filters%5B%5D=.go#diff-e961856e240d840bac6a3c12c6cd7496R75

Reply:
I think we should set it to the same plan what was used by the user when installing Toolchain operator. But yeah.. for now let's set it to Automatic for CRW: https://github.com/codeready-toolchain/toolchain-operator/pull/30/commits/a05321605e3500ebb56f58b82548cddb4f37aabd